src/main/java/account/SavingAccount.java

```diff
@@ -0,0 +1,100 @@
+package account;
+
+import interest.InterestCalculator;
+import interest.SavingInterest;
+import exception.account.BelowTargetException;
+import exception.account.InsufficienBalancetException;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.math.BigDecimal;
+
+@Getter
+@Setter
```

Comment:
@Setter는 public으로 선언된 객체에서 사용했을 경우 모든 곳에서 해당 객체의 필드 값을 변경할 수 있어 일관성을 유지하기 어려울 수 있습니다. 또한

```suggestion
@Setter(AccessLevel.NONE)
```
으로 해당 클래스 안에서만 사용할 수 있도록 해놓아도, 값을 변경하는 의도가 잘 드러나지 않을 수 있습니다. 

관련해서 setter의 대안으로 사용할 수 있는 방법을 찾아보시면 좋을 것 같습니다!